README.md

```diff
@@ -70,22 +70,167 @@ source for the project.  Packaging and cataloging are covered below.
 
 # Using The BrainIO Python Tools
 
-There are three main ways to use the Python BrainIO tools:
-* To access and analyze stimuli and data that are already packaged and cataloged.
+## Basic API
+
+The most commonly used methods of the BrainIO API are:  
+
+* **List Catalogs**: `brainio.list_catalogs()` lists the identifiers of all the BrainIO catalogs installed in your current Python environment.  
```

Comment:
I would put this last (or not include here at all), this is the least intuitive and likely least used out of all three to the standard user.

Reply:
I'm trying to change the "least used" part.  I want people to be more aware of the central role of catalogs.  That way, instead of filing a bug saying, "Why isn't my assembly loading?" they'll call `list_catalogs` first and realize that they don't have the right catalog installed.

Reply:
I would advocate to rather make things as automated as possible so that users do not even have to be aware of catalogs. I know this is a central component of brainio and important for other *libraries/devs* using it, but I don't think *users* should have to worry about it